launch_testing/launch_testing_examples/launch_testing_examples/check_msgs_launch_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from threading import Event
+from threading import Thread
+import unittest
+
+import launch
+import launch.actions
+import launch_ros.actions
+import launch_testing.actions
+import launch_testing.markers
+import pytest
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import String
+
+
+@pytest.mark.launch_test
+@launch_testing.markers.keep_alive
+def generate_test_description():
+    path_to_test = os.path.dirname(__file__)
+
+    return launch.LaunchDescription([
+        launch_ros.actions.Node(
+            executable=sys.executable,
+            arguments=[os.path.join(path_to_test, 'talker.py')],
```

Comment:
minor: I wonder if we can use one of the other example nodes here (instead of creating our own talker).

Reply:
I originally had the ``talker`` from ``demo_nodes_cpp``. I made a new talker to avoid dependencies, but I guess we can add it here safely. Will change it

Reply:
Changed to use ``demo_nodes_cpp``